pyinteraph/path_analysis.py

```diff
@@ -262,12 +265,18 @@ def get_graph_from_paths(paths):
                 graph[node1][node2]["e_weight"] += inc
```

Comment:
Check the indentation (I don't know if there is a double indentation here or it's just GitHub failing to show a single indentation properly).

Reply:
I checked the indentation level, it's working as intended